### ISSUE TITLE
Authorization ignored on Input Arguments

### ISSUE THREAD

**Issue by cotzo** [graphql-dotnet/authorization#16]
When using AuthorizeWith on the Fields that are used for query, everything works fine but it is ignored when set on arguments that implement InputObjectGraphType. Authorization of fields in the argument is not implemented?

 ```
public class AccountInputTypeCreate : InputObjectGraphType<Account>
    {
        public AccountInputTypeCreate()
        {
            Field(x => x.Name);
            Field(x => x.Value, true).AuthorizeWith(ClaimsEnum.ValueAccess.ToString());
        }
    }
```

**Comment by joemcbride** [comment]
Correct, the validation rule does not currently check Input objects.  You want to restrict users from providing a field on input?

**Comment by cotzo** [comment]
Hi Joe. That's correct. You have any easy way to do it? Thanks

**Comment by joemcbride** [comment]
I believe it should be able to be done by adding another match on `ObjectField`.  Up to doing a PR for it?

https://github.com/graphql-dotnet/authorization/blob/8ec6cd84854d293f9c250daf79c60dc91a5907d0/src/GraphQL.Authorization/AuthorizationValidationRule.cs#L29

**Comment by cotzo** [comment]
#17

**Comment by cotzo** [comment]
Fixed in ver 1.1.26
